source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEmployeesForEvaluationCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿using CommonJobs.Application.Evaluations.EmployeeSearching;
+using CommonJobs.Utilities;
+using CommonJobs.Domain;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.Evaluations
+{
+    /// <summary>
+    /// Command for getting the list of employees to populate the evaluation generation screen
+    /// </summary>
+    public class GetEmployeesForEvaluationCommand : Command<List<EmployeeEvaluation>>
+    {
+        private string _period;
+
+        public GetEmployeesForEvaluationCommand(string period)
+        {
+            _period = period;
+        }
+
+        public override List<EmployeeEvaluation> ExecuteWithResult()
+        {
+            RavenQueryStatistics stats;
+            IQueryable<Employee_Search.Projection> query = RavenSession
+                .Query<Employee_Search.Projection, Employee_Search>()
+                .Statistics(out stats)
+                .Customize(x => x.WaitForNonStaleResultsAsOfLastWrite());
+
+            query = query.Where(x => x.IsActive);
```

Comment:
Then, would it make sense to reuse the same Employee searches that were already available?